src/main/java/dk/runs/runners/rest/RunController.java

```diff
@@ -26,9 +26,10 @@ public Run getRun(@PathVariable int id){
     }
 
     @PostMapping(path = "/createRun")
```

Comment:
we do not use verbs in the path, create is defined by a post method so we just use /runs but a new method post